lib/dfe/analytics/entity_table_check_job.rb

```diff
@@ -4,25 +4,26 @@
 
 module DfE
   module Analytics
-    # Reschedules to run every 24hours
-    class EntityTableCheckJob < AnalyticsJob
-      WAIT_TIME = Date.tomorrow.midnight
+    # Reschedules with sidekiq_cron to run every 24hours
+    class EntityTableCheckJob
```

Comment:
Analytics job contains the DfE analytics queue as well as the retry config, could we not keep this.

Also my knowledge on this isn't great, but we we necessarily need to include `Sidekiq::Worker` ? Is it required for cron scheduling ?

Reply:
My initial thinking was that we would need to include it if services do not have Sidekiq installed as an active record adapter, but from what I can see, this is now the standard Sidekiq setup and a check of the services - they all have it installed like this so I can remove the include and also revert to inheriting from AnalyticsJob